cms/settings/dev.py

```diff
@@ -71,6 +71,7 @@
 ENABLE_DJANGO_DEFENDER = False
 
 # Auth
+SERVICE_AUTH_TOKEN = "WT1234AB-456C-789D-1E2F-90GH1234I567"  # noqa: S105 - Dummy token for local development
```

Comment:
Note, this is a dummy dev token that matches what is configured in the auth stub.

Reply:
@MaciekBaron that change is no longer needed due to [`6ea5deb` (#295)](https://github.com/ONSdigital/dis-wagtail/pull/295/commits/6ea5deba2cd43596b8adc2e49ca9adb9c3c4bc06)

@RealOrangeOne update the code comment.